sdk/storage/azdatalake/internal/path/models.go

```diff
@@ -28,6 +29,9 @@ func FormatDeleteOptions(o *DeleteOptions, recursive bool) (*generated.LeaseAcce
 	if o == nil {
 		return nil, nil, deleteOpts
 	}
+	if o.Paginated != nil {
+		deleteOpts.Paginated = o.Paginated
+	}
```

Comment:
You can directly do, `deleteOpts.Paginated = o.Paginated` instead of checking if it is nil or not.

Reply:
Made the change